Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -889,13 +920,27 @@ def enforcement_boundary_create_command(
         providers=providers,  # type: ignore
         ingress_services=[{"port": port, "proto": proto}],
     )
-    enforcement_boundary = client.enforcement_boundaries.create(enforcement_boundary_rule)  # type: ignore
-
-    enforcement_boundary_json = enforcement_boundary.to_json()
-
-    readable_output = prepare_enforcement_boundary_create_output(
-        enforcement_boundary_json
-    )
+    enforcement_boundary_json = ""
+    try:
+        enforcement_boundary = client.enforcement_boundaries.create(enforcement_boundary_rule)  # type: ignore
+        enforcement_boundary_href = enforcement_boundary.href
+        enforcement_boundary_json = enforcement_boundary.to_json()
+        enforcement_boundary_json["href"] = enforcement_boundary_href
+        enforcement_boundary_json["already_exists"] = False
+        readable_output = prepare_enforcement_boundary_create_output(
+            enforcement_boundary_json
+        )
+    except IllumioException:
+        enforcement_boundaries = client.enforcement_boundaries.get(params={"name": name})
+        for enforcement_boundary in enforcement_boundaries:
+            enforcement_boundary_json = enforcement_boundary.to_json()
+            enforcement_boundary_href = enforcement_boundary.href
+            enforcement_boundary_json["href"] = enforcement_boundary_href
```

Comment:
The HREF should already be set in the dict output from `to_json()`. If there's a bug here, can you please send me a code snippet that reproduces it?

Reply:
You can try with the following one,
`enforcement_boundary = EnforcementBoundary.build(name="Trial-101", consumers=["ams"],
                                                 providers=["ams"], ingress_services=[{"port": 8443, "proto": "TCP"}])
result = pce.enforcement_boundaries.create(enforcement_boundary)
print(result.to_json())`

Reply:
Running that code as far back as v1.0.0 I get

```
{'href': '/orgs/1/sec_policy/draft/enforcement_boundaries/41', 'ingress_services': [{'port': 8443, 'proto': 6}], 'providers': [{'actors': 'ams'}], 'consumers': [{'actors': 'ams'}], 'name': 'Trial-101', 'created_at': '2022-10-11T15:32:36.931Z', 'updated_at': '2022-10-11T15:32:36.962Z', 'update_type': 'create', 'created_by': {'href': '/users/1'}, 'updated_by': {'href': '/users/1'}, 'caps': ['write', 'provision']}
```

which looks fine to me. Which version of the library are you using?

Reply:
We've found some discrepancies in the Illumio SDK version in the dev and XSOAR instance. However we've noticed that if we're using the SDK version 1.1.1 then we're getting href but, in the case of the SDK version 1.0.0, we're not getting href.

Conclusion: We don't neet to explicitly assign the href as we're getting href from `to_json()`. Hence, removing the earlier change.